source/fuzz/transformation_add_bit_instruction_synonym.cpp

```diff
@@ -111,6 +116,48 @@ void TransformationAddBitInstructionSynonym::Apply(
   }
 }
 
+bool TransformationAddBitInstructionSynonym::IsInstructionOperandsHasSameSign(
```

Comment:
Probably better to rename the method to `IsInstructionSupported` if we are going to move those conditions into it.

Don't forget to adjust the docs.

Reply:
Done.